lib/common.sh

```diff
@@ -271,9 +272,9 @@ RESULT_STR=""
 export ANSIBLE_DISPLAY_SKIPPED_HOSTS=no
 
 # Sanity check for number of nodes
-if [ "${NUM_NODES}" -lt "$((NUM_OF_CONTROLPLANE_REPLICAS + NUM_OF_WORKER_REPLICAS))" ]; then
+if [ "$((NUM_NODES + NUM_REDFISHVMNODES))" -lt "$((NUM_OF_CONTROLPLANE_REPLICAS + NUM_OF_WORKER_REPLICAS))" ]; then
```

Comment:
This feels like it can become very confusing later. I get why it is this way, but I don't like it. One way to make it easier to understand I think, would be to rename `NUM_NODES` to something like `NUM_NORMAL_NODES` or `NUM_MIXED_NODES` or something like this.

Reply:
I agree, I will make separate PRs for this fix since this also need changes in capm3 and capm3 PR for this issue is already merged. Lets merge this one and open separate PRs for name changes.